Makefile.envs

```diff
@@ -1,5 +1,5 @@
 export PYVERSION ?= 3.10.2
-export PYODIDE_EMSCRIPTEN_VERSION ?= 2.0.27
+export PYODIDE_EMSCRIPTEN_VERSION ?= 2.0.28
```

Comment:
```suggestion
export PYODIDE_EMSCRIPTEN_VERSION ?= 2.0.27
```
There is a reason we are using Emscripten v2.0.27:
https://github.com/emscripten-core/emscripten/issues/16538

Reply:
I was worried this might cause problems - but there is some other clang or LLVM issue that causes a build error for version 2.0.27 and libhdf5.  It doesn't seem to happen for any other version of Emscripten.

Reply:
That is lucky! These LLVM issues are pretty worrisome. Hopefully I can get some time to work on this issue soon. I think it's the only blocker for us to get up to the most recent Emscripten release which would be great for us.